api/openapi.yaml

```diff
@@ -773,6 +1085,333 @@ components:
             tokens: "1234"
             model: "gpt-4-turbo"
         validationError: "meter not found for event"
+    Feature:
+      x-go-type-import:
+        path: github.com/openmeterio/openmeter/internal/credit
+      x-go-type: credit.Feature
+      type: object
+      description: |
+        A feature is a feature or service offered to a customer.
+        For example: CPU-Hours, Tokens, API Calls, etc.
+      required:
+        - name
+        - meterSlug
+      additionalProperties: false
+      properties:
+        id:
+          readOnly: true
+          description: |
+            Readonly unique ULID identifier of the feature.
+          type: string
+          example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
+        name:
+          description: |
+            The name of the feature.
+          type: string
+          example: AI Tokens
+        meterSlug:
+          description: |
+              The meter that the feature is associated with and decreases grants by usage.
+          type: string
+          example: tokens_total
+        meterGroupByFilters:
+          description: |
+            Optional meter group by filters. Useful if the meter scope is broader than what feature tracks.
+          type: object
+          additionalProperties:
+            type: string
+          example:
+            model: gpt-4
+        archived:
+          readOnly: true
+          description: |
+            If the feature is archived, it will not be used for grants or usage.
+          type: boolean
+          example: false
+    CreditLedgerEntry:
+      x-go-type-import:
+        path: github.com/openmeterio/openmeter/internal/credit
+      x-go-type: credit.LedgerEntry
+      type: object
+      description: |
+        Credit ledger entry.
+      required:
+        - id
+        - type
+        - time
+      properties:
+        id:
+          description: |
+            Readonly unique ULID identifier of the ledger entry.
+          readOnly: true
+          type: string
+          example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
+        type:
+          $ref: "#/components/schemas/CreditLedgerEntryType"
+        time:
+          description: |
+            The time the ledger entry was created.
+          type: string
+          format: date-time
+          example: "2023-01-01T00:00:00Z"
+        amount:
+          description: |
+            The amount to apply. Can be positive or negative number. If applicable.
+          type: number
+          example: 100
+        from:
```

Comment:
Maybe, instead of from/to can we use `usage_period.start` and `usage_period.end` or something that reflects that this is a metadata for the grant usage entry?